packages/credential-providers/src/fromNodeProviderChain.ts

```diff
@@ -2,11 +2,9 @@ import { defaultProvider, DefaultProviderInit } from "@aws-sdk/credential-provid
 import type { AwsCredentialIdentityProvider } from "@smithy/types";
 
 /**
- * This is the same credential provider as {@link defaultProvider|the default provider for Node.js SDK},
+ * This is the same credential provider as [the default provider for Node.js SDK](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-credential-providers/#fromnodeproviderchain),
```

Comment:
```suggestion
 * This is the same credential provider as {@link https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-credential-providers/#fromnodeproviderchain|the default provider for Node.js SDK},
```

https://tsdoc.org/pages/tags/link/